tests/scripts/thread-cert/Cert_9_2_09_PendingPartition.py

```diff
@@ -113,19 +143,28 @@ def test(self):
         self.nodes[COMMISSIONER].send_mgmt_pending_set(
             pending_timestamp=30,
             active_timestamp=210,
-            delay_timer=500000,
-            channel=20,
-            panid=0xafce,
+            delay_timer=1000000,
+            channel=CHANNEL_SECOND,
+            panid=PANID_INIT,
         )
         self.simulator.go(5)
 
         self.nodes[LEADER].remove_allowlist(self.nodes[ROUTER1].get_addr64())
         self.nodes[ROUTER1].remove_allowlist(self.nodes[LEADER].get_addr64())
-        self.simulator.go(140)
+        self.nodes[ROUTER2].set_preferred_partition_id(1)
+        self.simulator.go(250)
 
         self.assertEqual(self.nodes[ROUTER1].get_state(), 'router')
         self.assertEqual(self.nodes[ROUTER2].get_state(), 'leader')
 
+        self.nodes[ROUTER2].commissioner_start()
+        self.simulator.go(3)
+        self.nodes[ROUTER2].send_mgmt_active_set(
+            active_timestamp=15,
```

Comment:
nit: define constants for active timestamps and pending timestamps ?

Reply:
added more consts